--- a/Makefile	2017-07-08 23:40:42.000000000 +0200
+++ b/Makefile	2020-08-16 11:48:57.471759561 +0200
@@ -3,7 +3,7 @@
 endif
 
 ifndef LIBDIR
-	LIBDIR=/usr/lib/
+	LIBDIR=/lib/
 endif
 
 DPKG := $(shell dpkg-buildflags --version 2> /dev/null)
@@ -31,19 +31,19 @@
 
 
 PROGRAM = helm
-BIN     = $(DESTDIR)/usr/bin
+BIN     = $(DESTDIR)/bin
 BINFILE = $(BIN)/$(PROGRAM)
 LV2     = $(DESTDIR)/$(LIBDIR)/lv2/$(PROGRAM).lv2
 VSTDIR  = $(DESTDIR)/$(LIBDIR)/lxvst
 VST     = $(VSTDIR)/$(PROGRAM).so
-SYSDATA = $(DESTDIR)/usr/share/$(PROGRAM)
+SYSDATA = $(DESTDIR)/share/$(PROGRAM)
 IMAGES  = $(SYSDATA)/icons
 PATCHES = $(SYSDATA)/patches
-MAN     = $(DESTDIR)/usr/share/man/man1/
-CHANGES = $(DESTDIR)/usr/share/doc/$(PROGRAM)/
-DESKTOP = $(DESTDIR)/usr/share/applications/
+MAN     = $(DESTDIR)/share/man/man1/
+CHANGES = $(DESTDIR)/share/doc/$(PROGRAM)/
+DESKTOP = $(DESTDIR)/share/applications/
 
-ICONS   = $(DESTDIR)/usr/share/icons/hicolor/
+ICONS   = $(DESTDIR)/share/icons/hicolor/
 ICON16  = images/helm_icon_16_1x.png
 ICON22  = images/helm_icon_22_1x.png
 ICON24  = images/helm_icon_24_1x.png
